ddtrace/writer.py

```diff
@@ -59,18 +62,16 @@ def _reset_worker(self):
             self._worker = AsyncWorker(
                 self.api,
                 self._traces,
-                self._services,
                 filters=self._filters,
                 priority_sampler=self._priority_sampler,
             )
 
 
 class AsyncWorker(object):
 
-    def __init__(self, api, trace_queue, service_queue, shutdown_timeout=DEFAULT_TIMEOUT,
+    def __init__(self, api, trace_queue, shutdown_timeout=DEFAULT_TIMEOUT,
```

Comment:
We should probably go for `service_queue=None` here to keep backwards compatibility with the `AsyncWorker()` API.

Reply:
Could do a:

```python
if service_queue:
    deprecation()
```